src/automatic-annotations/shared/column-renderers/AnnotationCovered.tsx

```diff
@@ -0,0 +1,35 @@
+import { ExpandableList } from 'franklin-sites';
+
+import { ARBAAPIModel } from '../../arba/adapters/arbaConverter';
+import { UniRuleAPIModel } from '../../unirule/adapters/uniRuleConverter';
+
+const AnnotationCovered = ({
+  mainRule,
+}: Partial<UniRuleAPIModel | ARBAAPIModel>) => {
+  const annotations = new Set<string>();
+
+  for (const annotation of mainRule?.annotations || []) {
+    if ('keyword' in annotation) {
+      annotations.add('keyword');
+    } else if ('proteinDescription' in annotation) {
+      annotations.add('protein name');
+    } else if ('gene' in annotation) {
+      annotations.add('gene name');
+    } else if (annotation.comment?.commentType) {
+      annotations.add(annotation.comment?.commentType.toLowerCase());
+    } else if (annotation.dbReference?.database === 'GO') {
+      annotations.add('GO (Gene Ontology) term');
+    } else {
+      // in case we're missing a case
+      console.warn(annotation); // eslint-disable-line no-console
+    }
+  }
+
+  return (
+    <ExpandableList descriptionString="annotations" displayNumberOfHiddenItems>
+      {Array.from(annotations)}
```

Comment:
As far as I can see the order of the annotations is random. Sorting could slightly improve the ability to compare rows.

Reply:
Not sure what would be more useful, right now it's the order of first occurence in the rule (maybe from more to less important for the curator creating the rule?); could order alphabetically, to have consistent ordering; or could order by number of occurence of said annotation?

Reply:
The current site takes the order from the rule so that would be the safe choice

Reply:
OK, so will keep that then